trulens_eval/trulens_eval/feedback.py

```diff
@@ -1557,6 +1557,24 @@ def groundedness_measure(self, source: str, statement: str) -> float:
                                        ] = _re_1_10_rating(line) / 10
                     i += 1
             return groundedness_scores, {"reason": reason}
+        elif isinstance(self.groundedness_provider, AzureOpenAI):
+            reason = ""
+            for i, hypothesis in enumerate(
+                    tqdm(statement.split("."),
+                         desc="Groundedness per statement in source")):
+                plausible_junk_char_min = 4
+                if len(hypothesis) > plausible_junk_char_min:
+                    score = self.groundedness_provider._doc_groundedness(
+                        premise=source, hypothesis=hypothesis
+                    )
+                    reason = reason + str.format(
+                        feedback_prompts.GROUNDEDNESS_REASON_TEMPLATE,
+                        statement_sentence=hypothesis,
+                        supporting_evidence="[Doc NLI Used full source]",
+                        score=score * 10,
+                    )
+                    groundedness_scores[f"statement_{i}"] = score
```

Comment:
how is this not breaking?

_doc_groundedness is only defined for huggingface

Reply:
Thanks for catching - this should be resolved.